guides/INSTALLING.md

```diff
@@ -255,14 +255,16 @@ Run all of the services in the [web journey](#web-journey), **but** change the c
 
 and also run the following:
 
-* [florence](https://github.com/ONSdigital/florence) - use: `$ make debug ENCRYPTION_DISABLED=true`
+* [florence](https://github.com/ONSdigital/florence) - use: `$ make debug ENCRYPTION_DISABLED=true ENABLE_HOMEPAGE_PUBLISHING=true`
 * [The-Train](https://github.com/ONSdigital/The-Train)
 * [dp-api-router](https://github.com/ONSdigital/dp-api-router)
 
 If you also want to run Florence with the ability to edit images on the homepage (for the Featured Content section), you will need to additionally run:
 
-* [dp-image-api](https://github.com/ONSdigital/dp-image-api)
-* [dp-image-importer](https://github.com/ONSdigital/dp-image-importer)
+* [dp-image-api](https://github.com/ONSdigital/dp-image-api) - use: `make debug ENCRYPTION_DISABLED=true`
+* [dp-image-importer](https://github.com/ONSdigital/dp-image-importer) - use: `make debug ENCRYPTION_DISABLED=true`
+* [dp-upload-service](https://github.com/ONSdigital/dp-upload-service) - use `make debug ENCRYPTION_DISABLED=true`
+* [dp-download-service](https://github.com/ONSdigital/dp-download-service)
```

Comment:
Sorry, may have been ambiguous in my last comment. The download service needs the encryption flag but the image api doesn't. 
```suggestion
* [dp-image-api](https://github.com/ONSdigital/dp-image-api)
* [dp-image-importer](https://github.com/ONSdigital/dp-image-importer) - use: `make debug ENCRYPTION_DISABLED=true`
* [dp-upload-service](https://github.com/ONSdigital/dp-upload-service) - use `make debug ENCRYPTION_DISABLED=true`
* [dp-download-service](https://github.com/ONSdigital/dp-download-service) - use: `make debug ENCRYPTION_DISABLED=true`
```